src/panels/config/automation/trigger/types/ha-automation-trigger-numeric_state.ts

```diff
@@ -29,7 +29,11 @@ export class HaNumericStateTrigger extends LitElement {
       inputBelowIsEntity?: boolean
     ) =>
       [
-        { name: "entity_id", required: true, selector: { entity: {} } },
+        {
+          name: "entity_id",
+          required: true,
+          selector: { entity: { multiple: true } },
```

Comment:
By making this `multiple`, the value that we pass for `entity_id` should be an array, we should make sure this is the case.